site/pages/api/create-disruption.api.ts

```diff
@@ -26,6 +26,9 @@ export const formatCreateDisruptionBody = (body: object) => {
                 disruptionEndDate: values[2],
                 disruptionEndTime: values[3],
                 disruptionNoEndDateTime: values[4],
+                disruptionRepeats: values[5],
+                disruptionDailyRepeatsEndDate: values[6],
```

Comment:
We just need disruptionRepeats and disruptionRepeatEndDate here

Reply:
done